dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNLoadInterfaceIIDM.h

```diff
@@ -143,6 +143,12 @@ class LoadInterfaceIIDM : public LoadInterface, public InjectorInterfaceIIDM {
     country_ = country;
   }
 
+#if _DEBUG_
+  powsybl::iidm::Load& getUnderlyingLoad() {
```

Comment:
Where is this used? we usually have an access to the network IIDM to retrieve the load

Reply:
Required for unit tests in multithreading. I didn't find another way

Reply:
Please see TestLoadInterface.cpp: 

  Load& load = network.getLoad("LOAD1");

Reply:
OK